docker-images/windows_base/Dockerfile

```diff
@@ -11,7 +11,9 @@ ADD https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963
 RUN start /wait C:\vc_redist.x64.exe /quiet /norestart
 
 # Install chocolatey
-RUN @powershell -NoProfile -ExecutionPolicy unrestricted -Command "(iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))) >$null 2>&1"
+# which needs TLS1.2 for powershell too be enabled. it's 2020.
```

Comment:
We should probably have a ticket to check if that ever gets fixed on their end, and turn it off on our end too.

Reply:
More reading on this [here](https://blog.pauby.com/post/force-powershell-to-use-tls-1-2/). It is pointed out that technically, this has todo with .NET versions (time to upgrade).